src/main/java/com/faforever/client/api/FafApiAccessorImpl.java

```diff
@@ -257,6 +257,7 @@ public void uploadMod(Path file, ByteCountListener listener) {
   @Override
   public void uploadMap(Path file, boolean isRanked, ByteCountListener listener) {
     MultiValueMap<String, Object> multipartContent = createFileMultipart(file, listener);
+    multipartContent.add("metadata", "{\"isRanked\":\"" + isRanked + "\"}");
```

Comment:
Maybe a more elegant way to do this? Like using an object mapper as in the API (can't be autowired as it's not a bean in the client). Creating an extra class for this is a bit overkill.